propolis/src/hw/virtio/queue.rs

```diff
@@ -381,7 +381,7 @@ impl Chain {
         if (self.read_stat.bytes_remain as usize) < item_sz {
             return false;
         }
-        // SAFETY: We assume the mutable item reference we have received is
+        // Safety: We assume the mutable item reference we have received is
```

Comment:
This is minor, but I s/SAFETY/Safety for consistency regarding unsafe comments. If you prefer screamy case, that's fine too, just trying to be uniform.